aireview/src/aireview/prefect/deploy/manage_block.py

```diff
@@ -0,0 +1,112 @@
+import argparse
+import asyncio
+from azure.identity import DefaultAzureCredential
+from azure.keyvault.secrets import SecretClient
+from prefect import get_client, exceptions
+from prefect.blocks.system import Secret
+from prefect_snowflake import SnowflakeCredentials
+
+cli = SecretClient(
+    "https://kvdataprd.vault.azure.net/", credential=DefaultAzureCredential()
```

Comment:
**style**: The SecretClient is initialized globally. Consider moving it inside a function or class to avoid potential issues with long-running connections and to make the code more testable.